### ISSUE TITLE
Formula issue between sheets.

### ISSUE THREAD

**Issue by jinguangze** [Strumenta/article-python-excel#1]
It's failing when the formula try to calculate between sheets.
````
=IFERROR(VLOOKUP(A8,'CS paste ydays close'!$A$1:$Z$35,'CS paste ydays close'!H$1,FALSE),"")
````

**Comment by jinguangze** [comment]
[price_reference.xlsx](https://github.com/Strumenta/article-python-excel/files/6955647/price_reference.xlsx)

**Comment by alessiostalla** [comment]
Right. It's mentioned in the article that the code shown is simplistic and doesn't support cross-sheet references or, even worse, cross-file references. IIRC the format of these is not even standard – Excel uses one, Apple Numbers another.
